.database_generator/mangos_translation/lightshope_merge.py

```
@@ -0,0 +1,201 @@
+    except OSError:
+      pass
+
+  target_conn.executescript(converted)
+
+
+def _ensure_row_exists(conn: sqlite3.Connection, table: str, key_col: str, key_val):
+  conn.execute(f"INSERT OR IGNORE INTO {table} ({key_col}) VALUES (?)", (key_val,))
+
+
+def _merge_file_into_db(target_conn: sqlite3.Connection, sql_path: str, do_insert: bool, do_update: bool):
+  table, create_sql, columns, rows = _load_lightshope_table(sql_path)
```

Ruff (0.14.6)

Unpacked variable `create_sql` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)